bindings/c/test/fdb_api.hpp

```diff
@@ -613,6 +615,13 @@ class Tenant final {
 		tr.clear(toBytesRef(fmt::format("{}{}", tenantManagementMapPrefix, toCharsRef(name))));
 	}
 
+	static TypedFuture<future_var::KeyRef> getTenant(Transaction tr, BytesRef name) {
+		tr.setOption(FDBTransactionOption::FDB_TR_OPTION_READ_SYSTEM_KEYS, BytesRef());
+		tr.setOption(FDBTransactionOption::FDB_TR_OPTION_LOCK_AWARE, BytesRef());
+		tr.setOption(FDBTransactionOption::FDB_TR_OPTION_RAW_ACCESS, BytesRef());
+		return tr.get(toBytesRef(fmt::format("{}{}", tenantMapPrefix, toCharsRef(name))), false);
```

Comment:
This works for our current use case (existence checking), but we can similarly use the special keys (`\xff\xff/management/tenant_map`) that would align with other language bindings and not depend on internal implementation details. My recommendation would be to do that and not include a reference to the internal tenant map system keys at all.

Reply:
I guess also this should be returning a `TypedFuture<future_var::ValueRef>` rather than `KeyRef`, which if I understand correctly allows the result to be optional. Then we could check for the presence of the tenant by checking whether the optional is set.